projects/demo/src/modules/components/mobile-calendar/examples/6/index.html

```diff
@@ -0,0 +1,6 @@
+<div class="example">
+    <tui-mobile-calendar
+        [max]="max"
+        [min]="min"
+    />
+</div>
```

Comment:
The same comment

https://github.com/taiga-family/taiga-ui/pull/8264#discussion_r1698125359